Operational-Procedures/Tutorials/testing-a-csc.rst

```diff
@@ -0,0 +1,156 @@
+###########################
+Testing A CSC At The Summit
+###########################
+
+Overview
+^^^^^^^^
+
+This document decribes how to test a CSC, that is still under development, at the summit.
+
+Prerequisites
+^^^^^^^^^^^^^
+
+- Announcements are to be made in the summit-testing-annoucements Slack channel
+- People who participate in testing or infrastructure support/development must be actively monitoring that channel
+- All announcements for all work that may affect the summit DDS network or its infrastructure are required
+- Access to LOVE is required as well as access to the azar2 node with Docker rights.
+- If the CSC under test already is deployed at the summit, access to ArgoCD is required as well.
+
+Post-Conditions
+^^^^^^^^^^^^^^^
+
+- The CSC under test has been tested and/or debugged.
+- The CSC under test is not running anymore.
+- If a CSC, that already was deployed at the summit, was stopped during this tutorial, it is started again.
+
+Tutorial Steps
+^^^^^^^^^^^^^^
+
+Stopping the production CSC
+---------------------------
+
+The first thing to do, is to check and verify that the CSC under test currently is not running in the production environment at the summit.
+To do so, login to LOVE and open the CSCSummary+CSCG view.
+That view displays all deployed CSCs and informs about the statuses of the CSCs.
+If the CSC, that you are going to test, is in STANDBY (S) or ENABLED (E) mode, you need to set it OFFLINE.
+This can be done by returning to the main LOVE page and selecting the MTQueue or ATQueue, depending on which group the CSC belongs to.
+From the list of Standard Scripts, select ``set_summary_state.py`` by clicking on the blue icon to the right of the script name.
+
+.. figure:: ./_static/love-standard-scripts.png
+    :name: love-standard-scripts
+
+In the popup, specify the configuration data for the script, for instance:
+
+::
+
+    data:
+      -
+        - DIMM:1
+        - OFFLINE
+        - current
+
+would set the DIMM CSC with index 1 OFFLINE.
+
+.. figure:: ./_static/love-configuring.png
+    :name: love-configuring
+
+Connect to the deploy-env container
+-----------------------------------
+
+Once you are sure that no CSC is running in the production environment (anymore), you can login via ssh to azar2.
+After logging is, the first thing to do is to check and verify that the develop container is running.
```

Comment:
"After logging is," -> "After logging in to azar2,"

I think you need to specify what "logging in" means, e.g. 

```
ssh azar2.cp.lsst.org
```

Reply:
Adopted, thanks.